internal/server/deputy_events.go

```diff
@@ -0,0 +1,57 @@
+package server
+
+import (
+	"github.com/gorilla/mux"
+	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/sirius"
+	"net/http"
+	"strconv"
+)
+
+type DeputyHubEventInformation interface {
+	GetDeputyDetails(sirius.Context, int) (sirius.DeputyDetails, error)
+	GetDeputyEvents(sirius.Context, int) (sirius.DeputyEventCollection, error)
+}
+
+type deputyHubEventVars struct {
+	Path          string
+	XSRFToken     string
+	DeputyDetails sirius.DeputyDetails
+	DeputyEvents sirius.DeputyEventCollection
+	Error         string
+	Errors        sirius.ValidationErrors
+}
+
+func renderTemplateForDeputyHubEvents(client DeputyHubEventInformation, tmpl Template) Handler {
```

Comment:
This function is almost identical to the other render function. Not necessarily a job for now, but it would be nice to refactor.